<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Erdems-MacBook-Pro.local" failures="1" tests="1" name="Surefire test" time="3.162" errors="0" timestamp="24 Jul 2019 18:46:25 GMT">
  <testcase classname="component.FakeTest" name="fail" time="3.149">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
	at component.FakeTest.fail(FakeTest.java:21)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- fail -->
</testsuite> <!-- Surefire test -->
